responsive-web-design/introduction-to-html-forms.md

```diff
@@ -0,0 +1,59 @@
+<div align="center">
+    <img src="../images/html5.jpg" alt="Logo" height="170" align="center">
+    <h1 align="center">Introduction to HTML Forms</h1>
+</div>
+
+## 📝 Description
+> HTML forms are used to collect user input. The user input is most often sent to a server for processing.
+
+## 📋 Prerequisites
+- [ ] [Introduction to HTML and HTML5](https://github.com/Kick-StartDev/web-development-basic-curriculum/blob/responsive-web-design/responsive-web-design/introduction-to-html-and-html5.md).
+- [ ] [Introduction to CSS and Basic CSS](https://github.com/Kick-StartDev/web-development-basic-curriculum/blob/responsive-web-design/responsive-web-design/introduction-to-css-and-basic-css.md).
+- [ ] [Introduction to CSS Colors](https://github.com/Kick-StartDev/web-development-basic-curriculum/blob/responsive-web-design/responsive-web-design/introduction-to-css-colors.md).
+
+## 🎯 Learning Objectives
+- [ ] Understand the basic structure of HTML forms.
+- [ ] Basic understanding of HTML form elements and their purpose.
+- [ ] Basic understanding of HTML form attributes and their purpose.
+- [ ] Basic understanding of HTML form input types and their purpose.
+- [ ] Basic understanding of HTML form validation and its purpose.
+
+## 📹 Video
+
+- Please watch the following videos:
+    - <a href="https://www.youtube.com/watch?v=2O8pkybH6po" target="_blank">HTML Forms</a>
+
+    [![Video](https://img.youtube.com/vi/2O8pkybH6po/0.jpg)](https://www.youtube.com/watch?v=2O8pkybH6po)
+
+## 🔧 Instructions
+- If you don't have an account on [freecodecamp](freecodecamp.org), please create one and familiarize yourself with the platform.
+Then complete all the challenges provided in the excerises section below and submit your work.
+
+## 🚀 Excersises
+- Please complete the following excersises:
+    - [ ] [Learn HTML Form by Building a Registration Form](https://www.freecodecamp.org/learn/2022/responsive-web-design/learn-html-form-by-building-a-registration-form/step-1)
```

Comment:
This link is not working

Reply:
@ahzia  Thank you very much for your review.
Done ✅